index.html

```diff
@@ -5,7 +5,47 @@
   <meta charset="UTF-8">
   <title>LMHT</title>
   <link type='text/css' href='css/style.css' rel='stylesheet'/>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.8.3/jquery.min.js" type="text/javascript"></script>
+<script src="js/jquery-scrolltofixed-min.js" type="text/javascript"></script>
+
+<script type="text/javascript">
+
+
+    $(document).ready(function() {
```

Comment:
You may want to consider moving this to its own file and including via a script tag, but if this just the first stab and you're going to iterate on it I'm ok with leaving it here.